tests/test_data/plant_catalog.xml

```diff
@@ -0,0 +1,291 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think that you can use much smaller test file to test the validator, I think a file that has just a few lines would be enough to test it.